templates/parachain/node/src/service.rs

```diff
@@ -415,7 +416,7 @@ pub async fn start_parachain_node(
 		request_receiver: paranode_rx,
 		parachain_network: network,
 		advertise_non_global_ips,
-		parachain_genesis_hash: client.chain_info().genesis_hash,
+		parachain_genesis_hash: client.chain_info().genesis_hash.encode(),
```

Comment:
```suggestion
		parachain_genesis_hash: client.chain_info().genesis_hash.clone().as_bytes().to_vec(),
```
This will preserve old wire format.